main/ertp/ERTP-Guide/Overview.md

```diff
@@ -0,0 +1,276 @@
+  the representation of an asset, made up of a
+  *[value](#value)* *[&](https://agoric.com/documentation/glossary/#value)*  (the
+  size of the amount) and a
+  *[brand](#brand)* *[&](https://agoric.com/documentation/glossary/#brand)* (the
+  kind of asset). You can think of these as the "how many" and the
+  "what" of an asset.
+- **[AmountMath](#amountmath)** **[&]((https://agoric.com/documentation/glossary/#amountmath)**:
+Methods to do math operations on `amount` objects.
+- **[Purse](#purse)** **[&](https://agoric.com/documentation/glossary/#purse)**: an
+  object for long term `amount` object storage. 
+- **[Payment](#payment)** **[&](https://agoric.com/documentation/glossary/#payment)**:
+  an object for transfering an `amount` object to another party.
+- **[Mint](#mint)** **[&](https://agoric.com/documentation/glossary/#mint)**: an
+  object that creates new `amount` objects of a specific brand. Each brand has
+  a one to one relationship with a mint and vice versa.
+- **[Issuer](#issuer)** **[&](https://agoric.com/documentation/glossary/#issuer)**: 
+an object linked to a single `mint` object that creates empty `purse` and
+`payment` objects for that mint's brand and operates on payments of
+that brand. 
+
+## Fungible and Non-Fungible Assets
+
+There are two kinds of assets,
+*[fungible](https://agoric.com/documentation/glossary/#fungible)* and
+*[non-fungible](https://agoric.com/documentation/glossary/#non-fungible)*. 
+
+Fungible assets are interchangeable. For example, if you have 100
+one-dollar bills and need to pay someone 5 dollars, it doesn't matter
+which five of your one-dollar bills you give them. 
+
+Non-fungible assets are of the same type, but are not interchangeable
+and specific items must be used. For example, you might have 100
+tickets. But someone wanting to buy a ticket from you will want one
+for a specific date, a specific theater, a specific time, and let's
+say a specific seat (these aren't General Admission tickets). 
+
+You can't just pull out a ticket at random and sell it to them, since
+each ticket will have at least one different value from all the
+others. This might also affect the price; you'll want to charge more
+for a Friday evening third row center ticket than a Wednesday matinee
+balcony ticket, even if it's for the same act.
+
+## Security properties
+
+ERTP `purse` objects have a `deposit` method which takes a `payment`
+object as its argument. It first checks that the `payment` object is
+genuine and the same asset type as the `purse` (after all, our quatloos currency is
+a very unstable asset, so we wouldn't want to let people deposit
+quatloos and get credit for simoleans). Note: quatloos and simoleans
+are both imaginary currencies. 
+
+If everything passes the checks, the asset moves from 
+the `payment` to the `purse`. If there's a problem, it throws an error.
+
+After a successful deposit, ERTP guarantees:
+- The `payment` object is empty.
```

Comment:
the payment object is deleted from the issuer's records

Reply:
This means that the issuer no longer recognizes it, and no longer has any amount associated with it.